### PR TITLE
[DOCS-132] Minor docs improvements

### PR DESCRIPTION
**Pull request by fonty1**
Fixes typos and updated docs structure, slightly. 

https://streamr.atlassian.net/browse/DOCS-132

https://streamr.atlassian.net/browse/DOCS-117

### CODE REVIEW

File: app/src/docs/components/Pages/Products/CreateProduct/index.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import React from 'react'
+import DocsHelmet from '$docs/components/DocsHelmet'
+import DocsLayout from '$docs/components/DocsLayout'
+import CreateProductContent from '$docs/content/products/CreateProduct.mdx'
+
+const CreateProduct = () => (
+    <DocsLayout>
+        <DocsHelmet pageTitle="Create a product" />
```
Comment (mondoreale) [inline app/src/docs/components/Pages/Products/CreateProduct/index.jsx:10]: "Create a product" vs ["Create a Product"](https://github.com/streamr-dev/streamr-platform/pull/1056/files#diff-1c40c1686e3b4e5154806f60ba0d64c0R174)

Comment (fonty1) [inline app/src/docs/components/Pages/Products/CreateProduct/index.jsx:10]:
```suggestion
        <DocsHelmet pageTitle="Create a Product" />
```

**Comment by mondoreale** [comment]
@fonty1, anything blocking a merge here?

**Comment by fonty1** [comment]
> @fonty1, anything blocking a merge here? (except conflicts)

@mondoreale I'll just need to take a quick look after I finish off these explainer videos this week, is it holding up anything for you?

**Comment by mondoreale** [comment]
@fonty1 not at all. Just checking. Good luck with them vids, sir!

**Pull request merged**